SequenceBrowser/MRML/vtkMRMLSequenceBrowserNode.cxx

```diff
@@ -292,13 +292,12 @@ void vtkMRMLSequenceBrowserNode::ReadXMLAttributes(const char** atts)
       }
       SetIndexDisplayMode(indexDisplayMode);
     }
-    else if (!strcmp(attName, "indexDisplayDecimals"))
```

Comment:
Could you make this backward compatible? (if number of decimals are specified then generate a format string accordingly)